### PR TITLE
feat: Introduce cli plugin for helm

### PR DESCRIPTION
**Pull request by iocanel**
The pull request introduces a CLI modules that is automatically registered as a plugin on projects using `quarkus-helm`.

The CLI contains commands:
- list
- install
- upgrade
- uninstall

The commands are expected to work on both single and multi module projects.

### CODE REVIEW

**Comment by Sgitario** [comment]
Many thanks for the pull request. I will start reviewing it by the end of today.

**Review by Sgitario — APPROVED** [review]
I've updated the pull request with a commit to fix some typos I saw.  Let me know if you're ok with it.
I'm approving these changes, but I will add some test coverage in the CI in a later pull request.

**Comment by iocanel** [comment]
@Sgitario: Thanks for the editing. Your changes seems fine!

File: cli/src/main/java/io/quarkiverse/helm/Lint.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import static io.quarkiverse.helm.HelmUtil.chartOf;
+
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
+import picocli.CommandLine.Command;
+
+@Command(name = "lint", sortOptions = false, mixinStandardHelpOptions = false, header = "Helm Lint")
+public class Lint extends ChartCommand {
```
Comment (Sgitario) [inline cli/src/main/java/io/quarkiverse/helm/Lint.java:13]: Great addition @iocanel !

**Pull request merged**